pom.xml

```diff
@@ -65,16 +65,6 @@
       <artifactId>jersey-client</artifactId>
       <version>2.25.1</version>
     </dependency>
-    <dependency>
-      <groupId>org.json</groupId>
-      <artifactId>json</artifactId>
-      <version>20180130</version>
-    </dependency>
-    <dependency>
-      <groupId>javax.activation</groupId>
-      <artifactId>activation</artifactId>
-      <version>1.1.1</version>
-    </dependency>
```

Comment:
Did this take care of the warning we were recieving due to duplicate dependencies?

Reply:
yep! they both were included twice